cupy/lib/_shape_base.py

```diff
@@ -2,6 +2,9 @@
 
 import cupy
 from cupy._core import internal
+from cupy._creation.from_data import array, asarray
+from cupy._manipulation.dims import expand_dims
+from cupy._manipulation.kind import asfarray
 
```

Comment:
Our coding guidelines prohibit importing non-module symbols unless there is a specific reason.
https://docs.cupy.dev/en/stable/contribution.html#coding-guidelines